src/libs/Navigation/AppNavigator/ModalStackNavigators.js

```diff
@@ -79,6 +81,31 @@ const DetailsModalStackNavigator = () => (
     </DetailsModalStack.Navigator>
 );
 
+const ParticipantsModalStackNavigator = () => (
+    <ParticipantsModalStack.Navigator
+        path={ROUTES.PARTICIPANTS}
+    >
+        <ParticipantsModalStack.Screen
+            name="Participants_Root"
+            component={ParticipantsPage}
+            options={{
+                ...defaultSubRouteOptions,
+                title: 'Participants',
+            }}
+        />
+        <DetailsModalStack.Screen
+            name="Details_Root"
+            headerShown={false}
+            path={ROUTES.REPORT_ID}
+            component={DetailsPage}
+            options={{
+                ...defaultSubRouteOptions,
+                title: 'ParticipantsParticipant',
```

Comment:
This will show in a browser tab as `ParticipantsParticipant` since `title` sets this text as the webpage title.

Reply:
Perhaps a better format would be

`Report > Participants > {display name of participant}`

But I'm not sure we need to figure this out right now and can maybe just leave out the title entirely.